### PR TITLE
Migrate staging/src/k8s.io/legacy-cloud-providers/aws/tags.go to structured logging

### PR DESCRIPTION
**Pull request by lovebaby979**
/kind Structured Logging migration

Fixes #
update aws tag log

**Does this PR introduce a user-facing change?**:
NONE

```release-note
 Structured Logging migration
```

```docs
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
```

### CODE REVIEW

File: staging/src/k8s.io/legacy-cloud-providers/aws/tags.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func (t *awsTagging) initFromTags(tags []*ec2.Tag) error {
 	}
 
 	if legacyClusterID == "" && newClusterID == "" {
-		klog.Errorf("Tag %q nor %q not found; Kubernetes may behave unexpectedly.", TagNameKubernetesClusterLegacy, TagNameKubernetesClusterPrefix+"...")
+		klog.ErrorS(err,"Tag legacy nor prefix not found; Kubernetes may behave unexpectedly.", "TagNameKubernetesClusterLegacy", TagNameKubernetesClusterLegacy, " TagNameKubernetesClusterPrefix", TagNameKubernetesClusterPrefix+"...")
```
Comment (shubheksha) [inline staging/src/k8s.io/legacy-cloud-providers/aws/tags.go:97]: I'd change the error message to `Cluster legacy and prefix tags  not found; Kubernetes may behave unexpectedly"

File: staging/src/k8s.io/legacy-cloud-providers/aws/tags.go [hunk 2]
```diff
@@ -170,7 +170,7 @@ func (t *awsTagging) readRepairClusterTags(client EC2, resourceID string, lifecy
 			continue
 		}
 		if actual == "" {
-			klog.Warningf("Resource %q was missing expected cluster tag %q.  Will add (with value %q)", resourceID, k, expected)
+			klog.InfoS("Resource was missing expected cluster tag .  Will add (with value)", "resourceID",  resourceID, "expectClusterTag", k, "addTag", expected)
```
Comment (shubheksha) [inline staging/src/k8s.io/legacy-cloud-providers/aws/tags.go:173]: `expectClusterTag` -> `expectedClusterTag`

**Review by shubheksha — CHANGES_REQUESTED** [review]
Minor changes but looks good otherwise 👍

File: staging/src/k8s.io/legacy-cloud-providers/aws/tags.go [hunk 3]
```diff
@@ -225,7 +225,7 @@ func (t *awsTagging) createTags(client EC2, resourceID string, lifecycle Resourc
 
 		// We could check that the error is retryable, but the error code changes based on what we are tagging
 		// SecurityGroup: InvalidGroup.NotFound
-		klog.V(2).Infof("Failed to create tags; will retry.  Error was %q", err)
+		klog.V(2).InfoS("Failed to create tags; will retry.","Error", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/aws/tags.go:228]: please ensure gofmt.
run hack/update-gofmt.sh and check hack/verify-gofmt.sh.
and use "err", err

**Pull request closed without merging**